src/org/linphone/LinphoneActivity.java

```diff
@@ -358,12 +358,14 @@ public static void setBackgroundColorTheme(Context context){
 		final SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
 		String background_color_theme=prefs.getString(context.getResources().getString(R.string.pref_theme_background_color_key), "default");
 		//set background color independent
-		if(background_color_theme.equals("Red")) {
-			topLayout.setBackgroundResource(R.drawable.background_theme_red);
-		}else if(background_color_theme.equals("Yellow")) {
-			topLayout.setBackgroundResource(R.drawable.background_theme_yellow);
-		}else{
+		if (topLayout != null){
+			if(background_color_theme.equals("Red")) {
+				topLayout.setBackgroundResource(R.drawable.background_theme_red);
+			}else if(background_color_theme.equals("Yellow")) {
+				topLayout.setBackgroundResource(R.drawable.background_theme_yellow);
+			}else{
 			topLayout.setBackgroundResource(R.drawable.background);
+			}
```

Comment:
This seems like a crash fix, not a Crashlytics fix. Should this be a separate PR?